src/csv/report.rs

```diff
@@ -426,17 +434,24 @@ fn nominal_plot(table: &[HashMap<String, String>], column: String) -> Vec<PlotRe
     }
 
     let mut plot_data = Vec::new();
-    for (k, v) in count_values {
-        let plot_record = PlotRecord { key: k, value: v };
+    for (k, v) in &count_values {
```

Comment:
Use map/collect instead of for loop.

Reply:
Done in https://github.com/rust-bio/rust-bio-tools/pull/139/commits/ac61316a59982a6185c7c44adf4632b3708ed41d.